datalad/distribution/create_sibling.py

```diff
@@ -163,6 +157,28 @@ def _create_dataset_sibling(
                 lgr.info(_msg + " Skipping")
                 return
             elif existing == 'replace':
+                remove = False
+                if path_children:
+                    has_git = '.git' in path_children
+                    _msg_stats = _msg \
+                                 + " It is %sa git repository and has %d files/dirs." % (
+                                     "" if has_git else "not ", len(path_children)
+                                 )
+                    from datalad.ui import ui
```

Comment:
It's fine to leave this delayed import, but it doesn't actually help in this case:

```sh
#!/bin/sh

python <<\EOF | grep datalad | grep ui
import sys
import datalad.distribution.create_sibling
for k in sorted(sys.modules.keys()):
    print(k)
EOF
```

```
datalad.ui
datalad.ui.base
datalad.ui.dialog
```

Reply:
ok, will move up